msteams-platform/resources/removing-tab-margins.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Removing tab margins in Microsoft Teams
+author: laujan
+description: 
+keywords: tab removing margins padding
+ms.topic: reference
+ms.author: lomeybur
+---
+
+# Removing all tab margins (2021 update)
+
+Microsoft Teams is removing the margins around all tabs. This will allow developers to build apps that look more native to Teams. This will also align with our [UI kit designs](~/tabs/design/tabs.md). Most apps already look better without the margins surrounding their experiences. However, some tabs are visually affected by this change and developers should make the necessary changes.
+
+:::image type="content" source="../../assets/images/tabs/remove-margins-tabs.png" alt-text="Tab wit and without margins" border="false":::
+
+## Timelines
+
+* 3/1 - Margins removed in [Public Developer Preview](~/resources/dev-preview/developer-preview-intro.md)
```

Comment:
For clarity, let's spell out instead of assuming MM/DD format

Reply:
Updated.